CRM/Xcm/MatchingEngine.php

```diff
@@ -1115,7 +1126,18 @@ protected function createDiffActivity($contact, $options, $subject, &$contact_da
     }
 
     // if there is one address attribute, add all (so the user can later compile a full address)
-    $address_parameters = array('street_address', 'country_id', 'postal_code', 'city', 'supplemental_address_1', 'supplemental_address_2');
+    $address_parameters = array('street_address');
```

Comment:
`array()` should be replaced by the short array syntax `[]` on code that is changed.